### PR TITLE
upd(git): `2.34.1` -> `2.35.1`

### PR DESCRIPTION
**Pull request by cat-master21**

### CODE REVIEW

File: packages/git/git.pacscript
Comment (Henryws) [inline packages/git/git.pacscript:6]: Keep the `replace` variable.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/git/git.pacscript
Comment (Henryws) [inline packages/git/git.pacscript:7]: Remove `${name}` from `breaks`

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/git/git.pacscript
Comment (cat-master21) [reply packages/git/git.pacscript:6]: see: https://github.com/pacstall/pacstall-programs/runs/5859071549?check_suite_focus=true#step:9:26 Pacstall fails to install because of it

File: packages/git/git.pacscript
Comment (D-Brox) [reply packages/git/git.pacscript:7]: This was fixed in 1.7.3, IIRC

**Comment by wizard-28** [comment]
Waiting on @Henryws's approval.

**Review by Henryws — APPROVED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**
